scripts.js

```diff
@@ -1,5 +1,5 @@
 var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period';
 var dinosaur = 'triceratops';
 var dinosaurUpperCased = dinosaur.toUpperCase();
-var textCharsAfter = text.replace ('Velociraptor', 'triceratops')
+var textCharsAfter = text.replace('Velociraptor', dinosaur)
 console.log(text.lenght/2);
```

Comment:
in this line you should log in console half of changed text. use substring method